README.md

```diff
@@ -4,6 +4,7 @@ Mocking
 [![Build Status](https://travis-ci.org/invenia/Mocking.jl.svg?branch=master)](https://travis-ci.org/invenia/Mocking.jl)
 [![Build Status](https://ci.appveyor.com/api/projects/status/la041r86v6p5k24x?svg=true)](https://ci.appveyor.com/project/omus/mocking-jl)
 [![codecov.io](http://codecov.io/github/invenia/Mocking.jl/coverage.svg?branch=master)](http://codecov.io/github/invenia/Mocking.jl?branch=master)
+[![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
```

Comment:
I think it is better to have this on a seperate line than the automatic tests/coverage
```suggestion
<br/>
[![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
```

Reply:
I don't think that is necessary but we should decide before merging

Reply:
I agree with Curt. 

@oxinabox tell me if you feel strongly we need this and I'll accept the change

Otherwise I'll merge tomorrow as is

Reply:
I think this belongs on the line with links to other staticish things like docs and maybe lifecycle badges.

Reply:
I don't think it belongs with things that are life and updating and tell me about tests

Reply:
Fair enough :) I've no problem with that if it's consistent with what's already there (which here it is) -- will make the change